_redirects.yml

```diff
@@ -569,6 +569,10 @@
 "/develop/remote-development/":
   - /go/telepresence/
 
-# Builders
-"/build/drivers/":
+# Build links
+"/build/":
+  - /go/builds/
```

Comment:
I see on Docker Desktop that we redirect to `https://docs.docker.com/go/builds` but I think it should be:

```suggestion
"/desktop/use-desktop/builds/":
  - /go/builds/
```

Right?

Reply:
yea makes more sense